res/photon/index.js

```diff
@@ -0,0 +1,6 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+import '../css/photon-components.css';
+import 'photon-colors/photon-colors.css';
```

Comment:
I think it would be nice to have `'../css/focus.css'` as well. Probably as first import. This is what gives the focus indicators to the buttons.